jung-api/src/main/java/edu/uci/ics/jung/graph/GraphConstants.java

```diff
@@ -0,0 +1,14 @@
+package edu.uci.ics.jung.graph;
+
+final class GraphConstants {
```

Comment:
Hmm.  I should consider making the common.graph.GraphConstants class (and at least some of the constants in it) public, as some of this is (as I'm sure you're aware) just a copy; that way you wouldn't have to redefine them here.

Reply:
Sounds good to me! I'd actually forgotten that I'd copied some constants over from `common.graph`, so that would be a good thing to log somewhere.

Reply:
Maybe add a comment to that effect?

Reply:
Okey dokey! Done. :)